kademlia/src/test/java/ingsw/group1/kademlia/NodeUtilsTest.java

```diff
@@ -15,7 +14,7 @@
 public class NodeUtilsTest {
     BitSet peerKey;
     BitSet resourceKey;
-    Peer<String> peer;
+    SMSPeer peer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4743286)